tests/rules/SyntaxErrorInQueryMethodSubclassedRuleTest.php

```diff
@@ -58,18 +58,18 @@ public function getExpectedErrors(): array
                 return [
                     [
                         <<<TEXT
-    Query error: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "with"
-    LINE 1: SELECT with syntax error GROUPY by x LIMIT 0
-                   ^ (42601).
+Query error: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "with"
+LINE 1: SELECT with syntax error GROUPY by x LIMIT 0
+               ^ (42601).
 TEXT
                         ,
                         12,
                     ],
                     [
                         <<<TEXT
-    Query error: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "freigabe1u1"
-    LINE 1: SELECT email adaid WHERE gesperrt freigabe1u1 FROM ada LIMIT...
-                                              ^ (42601).
+Query error: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "freigabe1u1"
+LINE 1: SELECT email adaid WHERE gesperrt freigabe1u1 FROM ada LIMIT...
+                                          ^ (42601).
 TEXT
```

Comment:
I see :+1: 

I wonder if these would be better in txt files, as the spacing is crappy here and in PHP 7.3+ herenowdoc change can backfire again :)

Reply:
its easier to maintain in a single file instead of distributing it all over. its good enough for now :)